chain/x/oracle/keeper/request.go

```diff
@@ -54,7 +54,8 @@ func (k Keeper) AddRequest(
 		validators[i] = validatorsByPower[i].GetOperator()
 	}
 
-	ctx.GasMeter().ConsumeGas(executeGas, "ExecuteGas")
+	// TODO: Remove this hardcode (and KeyEndBlockExecuteGasLimit param)!
+	executeGas := uint64(100000)
```

Comment:
User doesn't need to pay execute gas. WOW

Reply:
Yep. It's all part of the package!